ARTCC/High/Daventry CTA.txt

```diff
@@ -159,7 +166,8 @@ Daventry CTA                N053.06.26.000 W002.43.49.000 N053.04.43.000 W002.40
 Daventry CTA                N053.04.43.000 W002.40.01.000 N052.58.58.000 W002.33.14.000 ;Extra Point to align with neighbouring airspace
 Daventry CTA                N052.58.58.000 W002.33.14.000 N052.42.16.000 W002.13.43.000 ;Extra Point to align with neighbouring airspace
 Daventry CTA                N052.42.16.000 W002.13.43.000 N052.18.56.000 W001.47.05.000 ;Extra Point to align with neighbouring airspace
-Daventry CTA                N052.18.56.000 W001.47.05.000 N053.03.14.000 W002.49.35.000 
```

Comment:
If you can add the note to CTA 20 too that'd be good

" - coordinates differ from AIP to ensure lines match up with other airspace"